puente/ext.py

```diff
@@ -5,10 +5,10 @@
 
 
 @jinja2.contextfunction
-def _gettext_alias(context, text, *args, **kwargs):
+def _gettext_alias(context, *args, **kwargs):
```

Comment:
It's actually a different context and text. This is the environment context, not the msgctxt which gettext gets. This is totally a bug--I screwed up the code here. I'm not sure how it worked for anyone.